system-tests/dashboard/test-dashboard.js

```diff
@@ -0,0 +1,394 @@
+require("../_support/utils/ServicesUtil");
+const { Timeouts } = require("../_support/constants");
+
+describe("Dashboard", function() {
+  beforeEach(() => {
+    cy.visitUrl("dashboard");
+  });
+
+  afterEach(() => {
+    cy.window().then(win => {
+      win.location.href = "about:blank";
```

Comment:
Yeah, Philipp gave me a bit of an explanation on this this morning too. Something we need to keep around.